nautobot/docs/configuration/optional-settings.md

```diff
@@ -289,6 +289,19 @@ The number of days to retain logged changes (object creations, updates, and dele
 
 ---
 
+## CONFIG_CONTEXT_DYNAMIC_GROUPS_ENALBED
+
+Default: `False`
+
+Environment Variable: `NAUTOBOT_CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED`
+
+If `True`, it will be possible to apply Config Context objects to devices and vms via Dynamic Group membership. When set to `False` this behavior will not be available.
```

Comment:
```suggestion
If `True`, it will be possible to apply Config Context objects to Devices and Virtual Machines via Dynamic Group membership. When set to `False` this behavior will not be available.
```